L/LibCURL/LibCURL/build_tarballs.jl

```diff
@@ -0,0 +1,3 @@
+include("../common.jl")
+
+build_libcurl(ARGS, "LibCURL")
```

Comment:
I don't want to rebuild libcurl, nothing changed.  This file should be removed from this PR and added in a follow-up PR with `[skip ci]`.

Reply:
Is it cool if I leave the file in until the PR is ready to merge, just to make sure that CI passes (and that I don't have any bugs for the libcurl code path)? And then, once the PR is ready to merge, I'll remove the file from this PR, and then you can merge immediately afterwards?